test/e2e/persistent_volumes-disruptive.go

```diff
@@ -182,8 +182,6 @@ func testVolumeUnmountsFromDeletedPod(c clientset.Interface, f *framework.Framew
 	result, err = framework.SSH("mount| grep "+string(clientPod.UID), nodeIP, framework.TestContext.Provider)
 	Expect(err).NotTo(HaveOccurred())
 	Expect(result.Code).NotTo(BeZero())
-
-	framework.Logf("Volume mount detected on pod and written file is readable post-restart.")
```

Comment:
removed log message as volume is getting unmounted here so there is no file.

added success message as below.
_framework.Logf("Volume is unmounted on node %s", nodeIP)_